File: Python/Estrutura sequencial/medidas.py
# Variáveis
# Em Python não é necessário fazer esse passo...
a: float; b: float; c: float; areaQuadrado: float; areaTriangulo: float; areaTrapezio: float

# Entrada de dados
a = float(input("Digite a medida A: "))
b = float(input("Digite a medida B: "))
c = float(input("Digite a medida C: "))

# Artíficio matemático
areaQuadrado = a ** 2
areaTriangulo = (a*b)/2
areaTrapezio = (a+b)*c/2

# Saída de dados
print(f"Área do quadrado {areaQuadrado:.4f}, triângulo {areaTriangulo:.4f} e do trapézio {areaTrapezio:.4f}")

File: C++/Estrutura sequencial/medidas.cpp
// Biblioteca
#include <bits/stdc++.h>

using namespace std;

int main()
{
    // Variáveis
    double a, b, c, areaQuadrado, areaTriangulo, areaTrapezio;

    // Entrada de dados
    cout << "Digite a medida A: : ";
    cin >> a;

    cout << "Digite a medida B: ";
    cin >> b;

    cout << "Digite a medida c: ";
    cin >> c;

    // Algoritmo matemático
    areaQuadrado = pow(a, 2);
    areaTriangulo = (a*b)/2;
    areaTrapezio = (a+b)*c/2;

    // Saída de dados
    cout << fixed << setprecision(4); // Quantidade de casas decimais
    cout << "Area do quadrado = " << areaQuadrado << endl;
    cout << "Area do triangulo = " << areaTriangulo << endl;
    cout << "Area do trapezio = " << areaTrapezio;

    return 0;
}